conf/resources/data/sa/transitionedToCdcsAndiTSAMigratedLiabilityAndDebtFalse.json

```diff
@@ -0,0 +1,161 @@
+  "cesaData": {
+    "customer": {
+      "individual": {
+        "title": "Mr",
+        "firstName": "Ben",
+        "lastName": "Thomas",
+        "dateOfBirth": "2014-09-12",
+        "dateOfDeath": "2014-09-12",
+        "identifications": [
+          {
+            "idType": 73,
+            "idValue": "SA123545"
+          }
+        ]
+      }
+    },
+    "addresses": [
+      {
+        "addressType": "PPOB",
+        "rls": false,
+        "emailAddress": "joe@snailmail.com",
+        "postcodeHistory": [
+          {
+            "addressPostcode": "BN127ER",
+            "postcodeDate": "2022-05-22"
+          },
+          {
+            "addressPostcode": "BN129ER",
+            "postcodeDate": "2022-02-28"
+          },
+          {
+            "addressPostcode": "BN119ER",
+            "postcodeDate": "2001-01-31"
+          }
+        ]
+      }
+    ],
+    "signals": [
+      {
+        "signalType": "Transitioned to CDCS",
+        "signalValue": "True",
+        "signalDescription": "CESA Transitioned to CDCS signal type"
+      }
+    ],
+    "filings": [
+      {
+        "icrCategory": "string",
+        "icrDescription": "string",
+        "fromDate": "2014-09-12",
+        "toDate": "2014-09-12",
+        "dueDate": "2014-09-12"
+      }
+    ],
+    "charges": [
+      {
+        "chargeType": "In Year RTI Charge - Tax",
+        "mainType": "In Year RTI Charge (FPS)",
+        "mainTrans": "2000",
+        "subTrans": "1000",
+        "UTR": "73",
+        "taxPeriodFrom": "2022-02-06",
+        "taxPeriodTo": "2022-03-05",
+        "dueDate": "<DUE_DATE>",
+        "interestStartDate": "2022-03-05",
+        "tiebreaker": "string",
+        "creationDate": "2022-03-05",
+        "saTaxYearEnd": "2022-03-05",
+        "originalChargeType": "string",
+        "originalTieBreaker": "string",
+        "chargeReference": "XM123456789812",
+        "outstandingAmount": 3000.00,
+        "accruedInterest": 500.00
+      }
+    ]
+  },
+  "etmpData": {
+    "customer": {
+      "individual": {
+        "title": "Mr",
+        "firstName": "Ben",
+        "lastName": "Thomas",
+        "dateOfBirth": "2014-09-12",
+        "dateOfDeath": "2014-09-12",
+        "identifications": [
+          {
+            "idType": 73,
+            "idValue": "SA123545"
+          }
+        ]
+      }
+    },
+    "addresses": [
+      {
+        "addressType": "PPOB",
+        "rls": false,
+        "emailAddress": "joe@snailmail.com",
+        "postcodeHistory": [
+          {
+            "addressPostcode": "BN127ER",
+            "postcodeDate": "2022-05-22"
+          },
+          {
+            "addressPostcode": "BN129ER",
+            "postcodeDate": "2022-02-28"
+          },
+          {
+            "addressPostcode": "BN119ER",
+            "postcodeDate": "2001-01-31"
+          }
+        ]
+      }
+    ],
+  "signals": [
+        {
+          "signalType": "ITSA MigraTed",
+          "signalValue": "True",
```

Comment:
ITSA Migrated signals have dates, not booleans. Its only the CDCS signals that have dates